aopy/visualization/base.py

```diff
@@ -2430,4 +2430,72 @@ def overlay_sulci_on_spatial_map(subject, chamber, drive_type, theta=0, center=(
     filename = f'{subject.lower()}_{chamber.lower()}_{drive_type.lower()}_sulci.png'
     params_file = as_file(config_dir.joinpath(filename))
     with params_file as f:
-        overlay_image_on_spatial_map(f, drive_type, theta=theta, center=center, alpha=alpha, **kwargs)
\ No newline at end of file
+        overlay_image_on_spatial_map(f, drive_type, theta=theta, center=center, alpha=alpha, **kwargs)
+
+def plot_plane(ax, plane, scale=1.0, color='grey', alpha=0.15, resolution=100):
+    """
+    Plots a 3D plane on a given Matplotlib axis.
+
+    This function generates a square plane centered at the origin, applies a transformation 
+    using a given mapping (either a rotation matrix or a plane equation), and renders it in 3D space.
+
+    Args:
+        ax (mpl_toolkits.mplot3d.Axes3D): The Matplotlib 3D axis on which to plot the plane.
+        plane (numpy.ndarray): Specifies how the plane is transformed:
+            - If shape (3,3) or (4,4): Treated as a transformation matrix for rotation.
+            - If shape (4,): Treated as plane equation coefficients (A, B, C, D) for Ax + By + Cz + D = 0.
```

Comment:
can you describe in more detail what a "transformation matrix for rotation" is?
```suggestion
        plane (4-tuple or (3,3) or (4,4) matrix): Specifies how the plane is transformed:
            - If shape (3,3) or (4,4): Treated as a transformation matrix for rotating the plane z=0.
            - If shape (4,): Treated as plane equation coefficients (A, B, C, D) for Ax + By + Cz + D = 0.
```